makefile

```diff
@@ -29,9 +29,9 @@ VALUES :=  \
 	-DHSI_VALUE=8000000 \
 	-DUSE_FULL_LL_DRIVER \
 	-DPREFETCH_ENABLE=1
-CFLAGS := $(MCU) $(VALUES) $(INCLUDES) -O2 -Wall -fdata-sections -ffunction-sections
+CFLAGS = $(MCU) $(VALUES) $(INCLUDES) -O2 -Wall -fdata-sections -ffunction-sections
```

Comment:
No, this is intentional, and the actual fix. :smile: 

The difference is significant here:
- `:=` (and any subsequent `+=`): the expression that is assigned is evaluated when the assignment is found while parsing the makefile (immediate);
- `=` (and any subsequent `+=`): the expression that is assigned is evaluated when the assignment is used in a makefile rule (deferred).

Since a subsequent `+=` expression uses `$@`, which is only defined in the context of a rule, this assignment **must** use deferred evaluation or it will break.

Reply:
Ahh ok